internal/olm/operator/internal/configmap.go

```diff
@@ -26,87 +26,111 @@ import (
 	"github.com/operator-framework/operator-registry/pkg/registry"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/yaml"
 )
 
-// IsManifestDataStale checks if manifest data stored in the registry is stale
-// by comparing it to manifest data currently managed by m.
-func (m *RegistryResources) IsManifestDataStale(ctx context.Context, namespace string) (bool, error) {
-	pkgName := m.Pkg.PackageName
-	nn := types.NamespacedName{
-		Name:      getRegistryConfigMapName(pkgName),
-		Namespace: namespace,
+// getRegistryConfigMaps performs a List operation to get all ConfigMaps
+// labelled as belonging to an operator's registry created by operator-sdk.
+func (rr *RegistryResources) getRegistryConfigMaps(ctx context.Context, namespace string) ([]corev1.ConfigMap, error) {
+	list := corev1.ConfigMapList{}
+	opts := []client.ListOption{
+		client.MatchingLabels(makeRegistryLabels(rr.Pkg.PackageName)),
+		client.InNamespace(namespace),
```

Comment:
These changes shows are 0% covered by tests. Could we add a few of then?

Reply:
Added an integration test case.